seed/entry_slug.sql

```diff
@@ -0,0 +1,57 @@
+ALTER TABLE entry DROP COLUMN IF EXISTS "slug";
+ALTER TABLE entry ADD COLUMN "slug" VARCHAR(255) DEFAULT NULL;
+
+-- slugify function is from kdobson.net/2019/ultimate-postgresql-slug-function
```

Comment:
later I want to handle situations when there is no title, or there are duplicate titles.

Reply:
Thats interesting. What do you think of using the created date when there is no title?

Reply:
sounds good, then maybe i could append the created date with a number to handle multiple untitled entries same date? or a very specific timestamp..

Reply:
Hmm that might work. Lets put a pin in that for the time being and revisit it in another PR